lstchain/io/event_selection.py

```diff
@@ -28,6 +25,7 @@ class EventSelector(Component):
 
     finite_params = List(
 	help="List of parameters to ensure finite values",
+	default_value={"intensity", "length", "width"},
```

Comment:
Your default value is a set (works but is a bit strange)

Reply:
Ah! I will correct it.